managing-cf/usage-events-and-billing.html.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Usage Events and Billing
+---
+
+## Usage Events
+Usage events can be used by Cloud Foundry operators to construct billing information for apps and service instances.
+
+Despite being similar in name, usage events are different from regular Cloud Foundry events (`/vX/events`). Cloud Foundry events should NOT be used for billing purposes; They are recorded regardless of the action succeeding and are not guaranteed to be in the correct order.
+
+### App Usage Events
+
+App usage events provide information about when users create, delete and update apps. They also include information about the app to enable operators to bill based on resource usage.
+
+App Usage Events expire after 31 days by default. This is configurable using the cf-release manifest property `cc.app_usage_events.cutoff_age_in_days`.
+
+#### Endpoint
+
+`GET /v2/app_usage_events`
+
+For documentation on the response information, see the [api docs](http://apidocs.cloudfoundry.org/latest-release/app_usage_events/list_all_app_usage_events.html)
+
+#### Event States
+App Usage Events have the following valid states: 'STARTED', 'STOPPED', and 'BUILDPACK_SET'.
+
+Multiple 'STARTED' events can occur in a row, so as to indicate updates to the app - you can parse the response to determine the specific difference.  
+
+Multiple 'STOPPED' events cannot occur without 'STARTED' events in-between them.  
+
+The app usage events with the 'BUILDPACK_SET' state do not represent an actual application state; They instead signify that a buildpack has been used for the application. This can be used by operators who wish to charge for buildpack usage.
+
+
+### Service Usage Events
+
+Service usage events provide information about when users create, update, and delete service instances.
+
+Service usage events are not persisted indefinitely. They do not currently expire like App Usage Events, but will in future releases of Cloud Foundry.
+
+#### Endpoint
+
+`GET /v2/service_usage_events`
+
+For documentation on the response information, see the [api docs](http://apidocs.cloudfoundry.org/latest-release/service_usage_events/list_service_usage_events.html)
+
+#### Event States
+
+Service Usage events have the following valid states: 'CREATED', 'DELETED', and 'UPDATED'.  These signify the desired state of the service.
+
+#### Managed Service Instances vs User Provided Service Instances
+
+Service usage events include the `service_instance_type` field to distinguish between managed service instances (service instances backed by a broker) and user provided service instances. You will likely not need to consider user provided service instances for billing purposes.
+
+## Using Usage Events
+
+
+### Ordering Usage Events
+Order of events returned from the API will match the sequence of events that occurred in the system.
```

Comment:
"Order of events returned from the API are guaranteed to match the sequence of events that occurred in the system."